meson.build

```diff
@@ -0,0 +1,36 @@
+project(
+    'ddbus',
+    'd',
+    meson_version: '>=0.50',
+    version: '3.0.0',
+)
+
+project_soversion = 0
+
+d_mod_version = []
+
+src_inc = include_directories('source')
+
+prefix = get_option('prefix')
+include_dir = join_paths(prefix, get_option('includedir'), 'd', 'apk-polkit')
```

Comment:
nitpick: Use the `/` syntax.

```meson
include_dir = prefix / get_option('includedir') / 'd' / 'apk-polkit'
```

Reply:
Why?

Reply:
I think it's a bit easier to read, especially when you have big meson files. But as I said, this is just a nitpick.

Reply:
Ah, I much prefer join_paths personally, so I think I'll keep it this way, but thanks for noting it :)